app/views/report/_reports.haml

```diff
@@ -21,6 +21,10 @@
               = report.text
         .panel-body
           .content
+            %strong
+              = t("report.author_label")
+            %p
+              = report.reported_author.diaspora_handle
```

Comment:
`Reported_author` can be `nil` when the reported item has been deleted. This can happen without admin action, for example if the post owner deletes a spam-comment that has been reported, which would make this throw a NoMethodError.

`.try()` or `&.` with a fallback-label of "Item deleted"or something would be a good-enough solution here.

Reply:
The `if` at line 11 ensures that the item is available here, so this should be fine as is. Otherwise it goes to the `else` after line 38 and doesn't show the author anymore.

Reply:
Oh.